content/articles/icann-domain-validation.md

```diff
@@ -29,11 +30,13 @@ The link must be to `raa.name-services.com`, `www.enom.com`, `registrant-email-v
 
 ## Resend verification email
 
-If you are having trouble finding the email verification in your inbox, you may want to resend it. Navigate to your domain page and locate the ICANN pending verification warning at the top of the page. Then click on "Send ICANN RAA verification email"
+If you are having trouble finding the email verification in your inbox, please remember to check your spam. 
+
+If it isn't there, resend the email by navigating to your domain page, and locating the ICANN pending verification warning at the top of the page. Click **Send ICANN RAA verification email**.
 
 ![ICANN resend email verification](/files/raa-resend-email-verification.png)
 
-Once you have acted on the verification email, the warning above should not appear. If it does, it may be because our system hasn't refreshed the latest status yet. You can manually force a refresh by clicking on the "Refresh" button.
+Once the email is verified, the warning should disappear. If it doesn't, it may be because our system hasn’t refreshed the latest status yet. Click **Refresh** to manually force a refresh. 
 
 ## What happens when a domain is suspended?
 
```

Comment:
can add the centralnic ones as soon as we get them, or if  that's going to take a bit, we can push the major changes and then make a note to revisit this when we have those

Reply:
I think we should push the changes now and add CentralNic later. We're still figuring out the CentralNic process for this. I discovered today that a lot of it is very different.

Reply:
@itsalyse got the CentralNic nameservers: ns1.emailverification.info
ns2.emailverification.info